src/System.Management.Automation/engine/parser/PSType.cs

```diff
@@ -265,6 +265,39 @@ internal static void DefineCustomAttributes(EnumBuilder member, ReadOnlyCollecti
             }
```

Comment:
@IISResetMe, your last commit had 4 failures in `PowerShell-CI-windows`
Error occurred in test script 'D:\a\1\s\test\powershell\Language\Classes\scripting.Classes.inheritance.tests.ps1'
```powershell
Object reference not set to an instance of an object.
at <ScriptBlock>, D:\a\1\s\src\powershell-win-core\bin\Release\net5.0\win7-x64\publish\Modules\Pester\4.10.1\Pester.psm1: line 1111
```
Error occurred in test script 'D:\a\1\s\test\powershell\Language\Classes\scripting.Classes.inheritance.tests.ps1'
```powershell
Object reference not set to an instance of an object.
at <ScriptBlock>, D:\a\1\s\src\powershell-win-core\bin\Release\net5.0\win7-x64\publish\Modules\Pester\4.10.1\Pester.psm1: line 1111
```
Error occurred in test script 'D:\a\1\s\test\powershell\Language\Classes\scripting.Classes.inheritance.tests.ps1'
```powershell
Object reference not set to an instance of an object.
at <ScriptBlock>, D:\a\1\s\src\powershell-win-core\bin\Release\net5.0\win7-x64\publish\Modules\Pester\4.10.1\Pester.psm1: line 1111
```
Error occurred in test script 'D:\a\1\s\test\powershell\Language\Classes\scripting.Classes.inheritance.tests.ps1'
```powershell
Object reference not set to an instance of an object.
at <ScriptBlock>, D:\a\1\s\src\powershell-win-core\bin\Release\net5.0\win7-x64\publish\Modules\Pester\4.10.1\Pester.psm1: line 1111
```